src/usts/ewma.cpp

```diff
@@ -39,7 +33,13 @@ auto ewma<Clock>::add(time_point time, double value) -> void {
 
 template<typename Clock>
 auto ewma<Clock>::get() const -> double {
-    return average;
+    return get(clock_type::now());
+}
+
+template<typename Clock>
+auto ewma<Clock>::get(time_point time) const -> double {
+    const auto alpha = calculate_alpha(time);
+    return (1 - alpha) * average;
```

Comment:
Hmm, probably I'm incorrect, please check, but it seems that alpha -> 1, when delta -> inf, so for `get` after long intervals of inactivity, result would -> 0.